README.md

```diff
@@ -18,12 +18,23 @@ $ npm install mbed-cloud-sdk
 
 `/node_modules/mbed-cloud-sdk` now contains:
 
-* `bundles` - minified browser scripts. 
-* `lib` - Node.js modules. 
+* `bundles` - minified browser scripts.
+* `lib` - Node.js modules.
 * `examples` - contains all examples.
 
 ## Documentation and examples
 
 See the full documentation and API reference at [https://cloud.mbed.com/docs/v1.2/mbed-cloud-sdk-javascript/](https://cloud.mbed.com/docs/v1.2/mbed-cloud-sdk-javascript/).
 
 Please refer to the examples folder for some node and web examples.
+
+## Contributing
+
+Mbed Cloud SDK for JavaScript is open source and we would like your help; there is a
+brief guide on how to get started in [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Licence
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
`license` or `licence`? We have both here :)

Can we put the license badge at the top and perhaps link to the SPDX license?

Reply:
I guess we better stick with the US spelling since it is the name of the file.